docs/book/src/clusterctl/commands/commands.md

```diff
@@ -1,19 +1,20 @@
 # clusterctl commands
 
-* [`clusterctl alpha rollout`](alpha-rollout.md)
-* [`clusterctl alpha topology plan`](alpha-topology-plan.md)
-* [`clusterctl backup`](additional-commands.md#clusterctl-backup)
-* [`clusterctl completion`](completion.md)
-* [`clusterctl config`](additional-commands.md#clusterctl-config-repositories)
-* [`clusterctl delete`](delete.md)
-* [`clusterctl describe cluster`](describe-cluster.md)
-* [`clusterctl generate cluster`](generate-cluster.md)
-* [`clusterctl generate provider`](generate-provider.md)
-* [`clusterctl generate yaml`](generate-yaml.md)
-* [`clusterctl get kubeconfig`](get-kubeconfig.md)
-* [`clusterctl help`](additional-commands.md#clusterctl-help)
-* [`clusterctl init`](init.md)
-* [`clusterctl move`](move.md)
-* [`clusterctl restore`](additional-commands.md#clusterctl-restore)
-* [`clusterctl upgrade`](upgrade.md)
-* [`clusterctl version`](additional-commands.md#clusterctl-version)
+* [`clusterctl alpha rollout`](alpha-rollout.md) Manage the rollout of a cluster-api resource.
+* [`clusterctl alpha topology plan`](alpha-topology-plan.md) List the changes to clusters that use managed topologies for a given input.
+* [`clusterctl backup`](additional-commands.md#clusterctl-backup) Backup Cluster API objects and all dependencies from a management cluster.
+* [`clusterctl completion`](completion.md) Output shell completion code for the specified shell (bash or zsh).
+* [`clusterctl config`](additional-commands.md#clusterctl-config-repositories) Display clusterctl configuration.
+* [`clusterctl delete`](delete.md) Delete one or more providers from the management cluster.
+* [`clusterctl describe cluster`](describe-cluster.md) Describe workload clusters.
+* [`clusterctl generate cluster`](generate-cluster.md) Generate templates for creating workload clusters.
+* [`clusterctl generate provider`](generate-provider.md) Generate templates for provider components.
+* [`clusterctl generate yaml`](generate-yaml.md) Process yaml using clusterctl's yaml processor.
+* [`clusterctl get kubeconfig`](get-kubeconfig.md) Gets the kubeconfig file for accessing a workload cluster.
+* [`clusterctl help`](additional-commands.md#clusterctl-help) Help about any command.
+* [`clusterctl init`](init.md) Initialize a management cluster.
+* [`clusterctl move`](move.md) Move Cluster API objects and all dependencies between management clusters.
```

Comment:
```suggestion
* [`clusterctl move`](move.md) Move Cluster API objects and all their dependencies between management clusters.
```

Reply:
Thank you for the feedback! All suggestions are fixed.